demux/demux_mkv.c

```diff
@@ -1539,6 +1539,15 @@ static int demux_mkv_open_video(demuxer_t *demuxer, mkv_track_t *track)
             extradata = track->private_data;
             extradata_size = track->private_size;
         }
+        if (fourcc1 == MKTAG('A', 'V', 'd', 'n') ||
+            fourcc2 == MKTAG('A', 'V', 'd', 'n') ||
+            fourcc1 == MKTAG('A', 'V', 'd', 'h') ||
+            fourcc2 == MKTAG('A', 'V', 'd', 'h'))
+        {
+            sh_v->codec = "dnxhd";
```

Comment:
What do you think about using `mp_set_codec_from_tag` instead? To avoid manual mapping of each codec we want to support.

Reply:
They seem to get detected as rawvideo when I use `mp_set_codec_from_tag`

Reply:
Hmm, ffmpeg has it defined, so should work.